### PR TITLE
Minor tweaks to inventory menus

### PR DESCRIPTION
**Pull request by codemime**
### Observable changes
- Don't show trailing punctuation marks in denials (look weird). See the first screenshot in #19952 (no dots are shown).
- Don't refresh erased window in `delwin` functor (provokes black rectangles). See the second screenshot in #19952 (the query's background isn't black).
- Faster and more natural toggling of multiple selections.

### CODE REVIEW

**Comment by keyspace** [comment]
Whether with this merged on top or without, #19952 shows ellipsis for me (using Terminus font). Not a big deal, I actually like it that way, but isn't that Terminus you're using, too?..

Anyway, might be locale-, terminal- or font-dependent.

The other part (query's background not black) works as intended.

File: src/output.cpp [hunk 1]
```diff
@@ -52,12 +52,9 @@ extern bool test_mode;
...
 void delwin_functor::operator()( WINDOW *w ) const
 {
-    if( w == nullptr ) {
-        return;
+    if( w != nullptr ) {
+        delwin( w );
     }
-    werase( w );
-    wrefresh( w );
-    delwin( w );
```
Comment (BevapDin) [inline src/output.cpp]: Other code may rely on this. The `WINDOW_PTR` is already used at several places which might require those `werase`, `wrefresh` calls.

Comment (codemime) [inline src/output.cpp]: I doubt that this `werase/refresh` really matters and that something relies on this.
The game should either restore the underlying background or redraw it completely. Replacing it with a black rectangle is meaningless. The only outcome of this is #15486.

**Comment by Coolthulhu** [comment]
Minor: picking up the object should not be required. Objects directly under the player can be disassembled without picking them up, extending that to nearby ones wouldn't be hard.

**Pull request merged**